src/modules/ROOT/pages/changelog/202303/02.json

```
@@ -27,80 +27,58 @@
     },
     {
       "what": [
-        "menu with 'more' options (delete, un-delete and permissions) to resources in {APP_NAME_COMPOSE}, {APP_NAME_ADMIN}, {APP_NAME_REPORTER}, {APP_NAME_WORKFLOW}."
+        "menu with 'more' options (delete, un-delete, and permissions) to resource lists in {APP_NAME_COMPOSE}, {APP_NAME_ADMIN}, {APP_NAME_REPORTER}, and {APP_NAME_WORKFLOW}."
       ],
       "why": [
-        "so the user doesn't have to scroll horizontally to access all resource button/options."
+        "so the user doesn't have to scroll horizontally to access all resource options."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/46f5455bd0ae67b9de2159f9d04b054a711832e1"
       ]
-    },
-    {
-      "what": [
-        "page layouts to namespace export resources."
-      ],
-      "refs": [
-        "https://github.com/cortezaproject/corteza/commit/4ad121747d136f76aa8701f07840c9b75c4aef37"
-      ]
     }
   ],
   "changed": [
     {
       "what": [
-        "{APP_NAME_ADMIN} role and user picker with vue-select."
-      ],
-      "why": [
-        "to be consistent with the rest of the pickers used in {PRODUCT_NAME}."
-      ],
-      "how": [
-        "by adding vue-select to role and member picker components."
-      ],
-      "refs": [
-        "https://github.com/cortezaproject/corteza/pull/1194"
-      ]
-    },
-    {
-      "what": [
-        "the implementation for reminders in {APP_NAME_ADMIN}."
+        "the way reminders in {APP_NAME_COMPOSE} look and behave. The new interface provides an easier way for the user to work with reminders"
```

```suggestion
        "the way reminders in {APP_NAME_COMPOSE} look and behave.",
        "The new interface provides an easier way for the user to work with reminders"
```

also, I think we can just squash these two sentences together